OutOfSchool/OutOfSchool.WebApi/Util/FakeImplementations/FakeImagesStorage.cs

```diff
@@ -1,13 +1,26 @@
 using System;
+using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Google.Api.Gax;
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
 using OutOfSchool.Services.Models.Images;
 using OutOfSchool.Services.Repository.Files;
+using Object = Google.Apis.Storage.v1.Data.Object;
 
 namespace OutOfSchool.WebApi.Util.FakeImplementations
 {
+    /// <summary>
+    /// Only for development purposes. Used as fake storage whenever no need to interplay with gcp storage.
+    /// </summary>
```

Comment:
Please wrap this and such sources with the `#if DEBUG # endif` directive so they won't be present in the production code.